modules/sandboxed-containers-deploying-workloads-with-kata-runtime-web.adoc

```diff
@@ -0,0 +1,77 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-deploying-workloads-with-kata-runtime-web_{context}"]
+= Deploying a workload in a sandboxed container using the web console
+
+{sandboxed-containers-first} installs the kata runtime as a secondary optional runtime on the cluster, and not as the primary runtime.
+
+To deploy a pod-templated workload in a sandboxed container, you must add the kata runtime manually to the workload's YAML file.
```

Comment:
Replace kata runtime.

Avoid using apostrophes. Use workload YAML file or YAML file of the workload.